src/tests/libecalc/integration/snapshots/test_all_consumer_with_time_slots_models/test_all_consumer_with_time_slots_models_results/all_consumer_with_time_slots_models_v3.json

```diff
@@ -654,8 +654,8 @@
                                 ]
                             },
                             "is_valid": [
-                                false,
```

Comment:
is_valid changes for the compressor stage results because the compressor stages operate within capacity, hence the compressor stage result is valid. The compressor train is still not able to meet the required discharge pressure, so is_valid on the compressor train is still false.